frontend/coprs_frontend/coprs/forms.py

```diff
@@ -655,6 +656,17 @@ class CoprForm(BaseForm):
         filters=[StringListFilter(), StripUrlSchemaListFilter()],
         validators=[wtforms.validators.Optional()],)
 
+    repo_priority = wtforms.IntegerField(
+        "The priority value of this repository",
+        description="""If there is more than one package to install from (e.g. vim from
+        Fedora repo and Copr repo), the repo with lower priority is picked.""",
+        render_kw={"placeholder": "Optional - integer, e.g. 22"},
+        validators=[
+            wtforms.validators.Optional(),
+            wtforms.validators.NumberRange(min=1, max=DEFAULT_REPO_PRIORITY_VALUE)],
```

Comment:
I think the `min` value looks reasonable but IMHO there should be no `max`. Someone might want to de-prioritize their repo and prefer the default repositories. I don't why would they do it but we should make it possible to do. If we want to have `max`, let's make it at least 999.